### PR TITLE
New Basemaps: Geometry Category

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

This PR is for the following sample categories:
- Geometry

Notes specific to this PR: 
- Project sample uses quite a different Basemap due to there not being a direct map for National Geographic so took a new image too
- Also in Project sample there was a stray, duplicate Java class that @sclaridge spotted and removed

### General comments on Basemap updates:

It updates `Map` and/or `Basemap` Constructors as per the new design. This may include the following changes depending on the sample:
- Replace `Basemap.Type` with `BasemapStyle`
- If a lat/long/zoom was being set in the `Map` constructor, this is replaced with a `Viewpoint` being set on the `MapView`.
- Any Readme Updates identified

In addition, the samples that include the above changes, or other services requiring an API Key going forward, have the API Key  set within the sample. This will then work in conjunction with the Gradle Task being added in PR 593.

Once 593 is approved and merged into `dev` we will merge those changes into these PRs and then update all the Samples to `100.10.0-xxxx` and do a round of testing of the Samples with all the API Key / Basemap changes at that time. (We have left them as 100.9.0 (in most cases) for now to minimize merge conflicts in the `build.gradle` file).

In the meantime, the updates listed above can be reviewed. Thanks!

### CODE REVIEW

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java
Comment (JonLavi) [inline geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]: I think wkid 3857 is WebMercator:
https://developers.arcgis.com/web-map-specification/objects/spatialReference/
So we could use the enum `SpatialReferences.getWebMercator()`:
https://developers.arcgis.com/java/latest/api-reference/reference/com/esri/arcgisruntime/geometry/SpatialReferences.html#getWebMercator()

However, I'm pretty sure web mercator is the default SR (or WGS84??), so might be worth leaving it out of the 'new Point` and seeing if it works?

File: geometry/geometry-engine-simplify/src/main/java/com/esri/samples/geometry_engine_simplify/GeometryEngineSimplifySample.java
Comment (JonLavi) [inline geometry/geometry-engine-simplify/src/main/java/com/esri/samples/geometry_engine_simplify/GeometryEngineSimplifySample.java]: A suggestion for plain english
```suggestion
      // enable geometry operations when the map is done loading
```

File: geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java
Comment (JonLavi) [inline geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java]:
```suggestion
      // create a map with the light gray basemap style
```

File: geometry/project/src/main/java/com/esri/samples/project/ProjectSample.java
Comment (JonLavi) [inline geometry/project/src/main/java/com/esri/samples/project/ProjectSample.java]: Same here, we could write `SpatialReferences.getWebMercator()`, and possibly even leave it out?
For the comment, maybe we should capitalise `Web Mercator` as it's an established term? Not 100% sure though
```suggestion
      // create a map with the Web Mercator spatial reference and a basemap style
      ArcGISMap map = new ArcGISMap(SpatialReferences.getWebMercator());
```

**Review by JonLavi — APPROVED** [review]
Just a few minor suggestions here a well! 👍

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java
Comment (jenmerritt) [reply geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]: Didn't work removing it. Have replaced it for `getWebMercator()`.  I also made a couple of other additional changes to this one - setting the `ViewPoint` on the `MapView` (which still seems to work correctly) and corrected the basemap style to `IMAGERY_STANDARD`. 

Probably one worth an extra check @Rachael-E

File: geometry/project/src/main/java/com/esri/samples/project/ProjectSample.java
Comment (jenmerritt) [reply geometry/project/src/main/java/com/esri/samples/project/ProjectSample.java]: I've tidied this up a bit

File: geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java
Comment (Rachael-E) [inline geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java]: this is a different scale value than was used in a previous PR https://github.com/Esri/arcgis-runtime-samples-java/pull/599/files#diff-8f2b646da59e98410b35da25b663995af9c16ab65fb85d1d6bea55aeff5de1df

We should try and be consistent with the values that we are swapping: the other values have been consistent between PRs for now :) 

if you haven't already seen it, you can refer to this to get an idea of what the exact replacement values would be, but rounding it down as you have been doing should be fine for samples. https://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer

where e.g. an old Level of Detail 12 would now be a new scale of 144447.638572

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java
Comment (Rachael-E) [inline geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]: can remove unused import

**Review by Rachael-E — APPROVED** [review]
looks good @jenmerritt , just keen to ensure when we're replacing the Level of Detail parameter with scale we are keeping roughly close enough to the Level of Detail's related scale, I'll approve so as not to hold up assuming you can check into that value I questioned :)

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java
Comment (Rachael-E) [reply geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]: looks good!

File: geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java
Comment (jenmerritt) [reply geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java]: For reference, I've picked up on this point across the changes samples and will correct the scale vs level of detail as per the above link (but rounded)

**Comment by jenmerritt** [comment]
`dev` merged in (including API Key gradle tasks changes). Samples that have an API Key set have been tested running from the IDE.

Await merging these changes until release.

**Pull request merged**
